<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>289</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Clientes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BIENVENIDO CLIENTE</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblCliente">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>261</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
    <column>
     <property name="text">
      <string>CEDULA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOMBRE</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAgregarC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>AGREGAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/agregar-usuario.png</normaloff>../assets/images inventario/agregar-usuario.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditarC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>EDITAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/editar-codigo.png</normaloff>../assets/images inventario/editar-codigo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminarC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 79, 79);</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/borrar.png</normaloff>../assets/images inventario/borrar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegresarC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>VOLVER</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/flecha-hacia-atras.png</normaloff>../assets/images inventario/flecha-hacia-atras.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
